### ISSUE TITLE
[enhancement] "no swizzle" ship attribute

### ISSUE THREAD

**Issue by Amazinite** [endless-sky/endless-sky#1357]
http://steamcommunity.com/app/404410/discussions/0/359543951710299936/

**Comment by Elyssaen** [comment]
Here you go: https://github.com/endless-sky/endless-sky/compare/master...Elyssaen:feature/NoSwizzle

Format is:

```
ship "Trin Raja"
	sprite "ship/trin raja"
		"frame rate" 20
		rewind
	"no swizzle"
	attributes
		category "Interceptor"
		tags alien trin
		...
```

Only just did it, seems to work, but I'll review it a bit and put in a PR only when I'm fully happy.

**Comment by Amazinite** [comment]
Given that #2629 has been merged, this issue is effectively (v0.9.7) done.
